catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/ops/IcebergTableOps.java

```diff
@@ -42,8 +42,7 @@ public class IcebergTableOps implements AutoCloseable {
   public IcebergTableOps(IcebergConfig icebergConfig) {
     String catalogType = icebergConfig.get(IcebergConfig.CATALOG_BACKEND);
     if (!IcebergCatalogBackend.MEMORY.name().equalsIgnoreCase(catalogType)) {
-      // The method `get` will call the method `checkValue`, just check the values of the config
-      // options
+      // The method `get` will check the values of config options.
```

Comment:
You don't have to add the comments here, it's not necessary.

Reply:
Fixed.